easybuild/easyconfigs/c/CMake/CMake-3.4.3-foss-2016.04.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.4.3'
+
+homepage = 'http://www.cmake.org'
+description = """CMake, the cross-platform, open-source build system.
+ CMake is a family of tools designed to build, test and package software."""
+
+toolchain = {'name': 'foss', 'version': '2016.04'}
+
+source_urls = ['http://www.cmake.org/files/v%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = '--DCMAKE_USE_OPENSSL=1'
+
+dependencies = [
+    ('ncurses', '6.0'),
+    # OS dependency should be preferred (uncomment osdependencies line) if the os
+    # version is more recent then this version. It's nice to have an up to date openssl
+    # for security reasons
+    ('OpenSSL', '1.0.1p'),
```

Comment:
please comment out this dep, and uncomment the `osdependencies` line below, cfr. comment above